interfaces/nginx_route/v0/schemas/requirer.json

```diff
@@ -0,0 +1,176 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://canonical.github.io/charm-relation-interfaces/interfaces/nginx_route/schemas/requirer.json",
+    "type": "object",
+    "title": "`nginx-route` requirer schema",
+    "description": "The `nginx-route` root schema comprises the entire requirer databag for this interface.",
+    "default": {},
+    "examples": [
+        {
+            "service-hostname": "example.test",
+            "service-name": "test-app",
+            "service-port": 8080,
+            "service-namespace": "test-namespace"
+        }
+    ],
+    "required": [
+        "service-hostname",
+        "service-name",
+        "service-port",
+        "service-namespace"
+    ],
+    "properties": {
+        "service-hostname": {
+            "$id": "#/properties/service-hostname",
+            "type": "string",
```

Comment:
Should this include a format like this? https://json-schema.org/understanding-json-schema/reference/string.html#hostnames

Reply:
Similar question for the others, should they have a format based on what is valid

Reply:
Yes, you are correct, updated. Thanks!
For other fields, I think they are all in basic types.